spec/views/posts/index_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe 'User Show', type: :feature do
+  before :each do
+    @user = User.create(name: 'User1', photo: 'https://picsum.photos/200', bio: 'This is a bio')
+    @post = Post.create(title: 'Post1', text: 'This is a post', author_id: @user.id)
+    @comment = Comment.create(text: 'This is a comment', author_id: @user.id, post_id: @post.id)
+  end
+
+  it 'displays the user name' do
+    visit user_path(@user)
+    expect(page).to have_content(@user.name)
+  end
+
+  it 'displays the user bio' do
+    visit user_path(@user)
+    expect(page).to have_content(@user.bio)
+  end
```

Comment:
- [ ] You have done a great job with your tests, however there are a few tests below that you have missed which are required 👇
- [ ] Tests that asserts you can see the user's profile picture.
- [ ] Tests that asserts you can see a post's title.
- [ ] Tests that asserts you can see some of the post's body.
- [ ] Tests that asserts you can see the first comments on a post.
- [ ] Tests that asserts you can see how many comments a post has.
- [ ] Tests that asserts you can see a section for pagination if there are more posts than fit on the view.
- [ ] Tests that redirects to the post's show page when you click on a post.